api/storage_service.cc

```diff
@@ -1565,11 +1565,13 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
         auto table_id = ctx.db.local().find_column_family(ks, table).schema()->id();
         auto force_str = req->get_query_param("force");
         auto force = service::loosen_constraints(force_str == "" ? false : validate_bool(force_str));
+        auto copy_str = req->get_query_param("copy");
+        auto copy = service::copy_tablet(copy_str == "" ? false : validate_bool(copy_str));
```

Comment:
What does it mean to copy tablet in the context of "move" tablet? If we're not really moving it, then perhaps it should not be part of the "move" API, it's confusing.

Reply:
We could add a new API to add a replica.

Reply:
Agree, it's not extremely elegant. Is this "move" API going to be "officially supported" one, i.e. -- not testing-only?

Reply:
> We could add a new API to add a replica.

This is what I wanted to avoid, because it's duplication of ALTER functionality :(

Reply:
> > We could add a new API to add a replica.
> 
> This is what I wanted to avoid, because it's duplication of ALTER functionality :(

But it would behave like move_table?copy=true, the new replica has to be provided by the user.